index.rst

```diff
@@ -4,9 +4,173 @@ Collimated Beam Projector Installation and ComCam Testing
+      :header-rows: 0
+
+      * - .. image:: images/one_per_amp_mask.png
+            :width: 300px
+        - .. image:: images/two_line_mask.png
+            :width: 300px
+
+   **Figure 5**: The one-pinhole-per-amp mask illuminated with a blue LED (left) and the design for the mask with two lines of pinholes of varying sizes (right).
+
+
+CBP Testing with ComCam
+=======================
+
+With heroic effort from the electronics, electrical, IT, and construction teams, we were able to install the CBP and laser, hook them up, and then take data with the CBP during the last four nights of the ComCam campaign.
+
+Here is a list of the major tests and milestones we were able to achieve:
+
+1. We were able to copoint the CBP and the Simonyi Telescope.
+2. We demonstrated that the copointing locations were repeatable from night to night even as the dome, TMA, and CBP had moved during the day.
+3. We swept through focus for all of our masks.
+4. We made small movements of the CBP and TMA to see how repeatable and accurate the movements were.
+5. We performed a copointing 'dance' where we moved the CBP and telescope at the same time and kept the CBP spots on ComCam.
+6. We swept through wavelengths in the g and r bands, to find our in-band, out-of-band, and band-edge throughput. We observed a red leak in the g band.
+7. We took images to investigate nonlinearity in the crosstalk.
+8. We took images while applying different AOS bending modes and saw how the spots moved.
+9. We looked at diffraction off the channel stops from red light.
+
+.. container:: figure
+
+   .. list-table::
+      :widths: 50 50
+      :header-rows: 0
+
+      * - .. image:: images/one_per_amp_image.png
+            :width: 300px
+        - .. image:: images/two_lines_image.png
+            :width: 300px
+
+   **Figure 6**: The one-pinhole-per-amp imaged on ComCam (left) and the two lines of pinholes of varying sizes imaged on ComCam (right).
+
+
+.. container:: figure
+
+   .. list-table::
+      :widths: 50 50
+      :header-rows: 0
+
+      * - .. image:: images/one_per_CCD_image.png
+            :width: 300px
+        - .. image:: images/filter_throughput.png
+            :width: 300px
+
+   **Figure 7**: The one-pinhole-per-CCD mask with a scaling so that the ghosts are clearly visible (left) and (right) the filter transmission curves for g and r band, calculated by doing photometry on images like the left. Photometry of spots with the filter in were divided by photometry of the spots with the filter removed.
+
+
+
+
+Summary and Next Steps
+======================
+
+Overall, the testing campaign with ComCam was very successful. We were able to demonstrate that the CBP works and to take the data that we need to design the masks we want for LSSTCam. We also took data to find out exactly what the relevant 
+
+We are currently in the process of analyzing the data. We are also working on software to make the process of data collection more automated during LSSTCam.
```

Comment:
Probably worth listing a couple main takeaways: magnification difference, evidence of r-band leakage, anything about the filter throughput?